README.md

```diff
@@ -42,7 +46,11 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 Create a For Loop that outputs the following string for each piece of fruit in the console. "I want to eat a [fruit]"
 
 ```js
-// write code here
+
+for(var i = 0; i<=foods.length; i++){
```

Comment:
I'll give you a :+1: because it would actually work -- but it shouldn't! You should use `i < foods[0].length` instead. Or, better yet:
```
var fruits = foods[0];
for(var i = 0; i < fruits.length; i++){
```